### DISCUSSION TITLE
Suggestions are welcome

### DISCUSSION THREAD

**Discussion by alboss** [alboss/goals#1] [General]
Got more to add? Submit your suggestions here.

**Comment by dereksheppard** [comment]
No Gonkulator?

**Reply by alboss** [reply]
Tragic oversight on my part! It's added. Along with an option to more easily see what's in all the arrays.

**Comment by rednbenji4** [comment]
Intentions:
After careful consideration I've decided to...
In order to demonstrate my superior capabilities, I have resolved to...
To reinforce the directorate guidance contribution to personal assigned tasks at hand, my actions will only slightly deviate from my original intentions in which I...

Action verbs:
crack out on
actuate
influence the workload of

What we measure:
widgets
fiscal responsibility
investments in beanie babies
gizmos developed

Ways not to say measurement:
within two standard deviations of the benchmark of
allotted by
in order to fulfill
valued according to

How we measure it:
quantifiable increments
the standard pie graph
12 parsecs
a Sagan
3 donkey-power

This is relevant because:
why not?
there's a snake in my boots.
importance and purpose can't always be inferred by management.
you can be the ripest, juiciest apple in the world, and there's still going to be somebody who loves windows.
people will only want to engage if it helps them solve problems.
it is creative and I believe in it.  Whatever THAT is.
it is not something you can actually predict. It is something that will be determined after the fact.
I am passionate about personalization.
it becomes irrelevant inside results closely connected to management's queries.


When it will be completed:
when Marion Crane takes a shower.
when ET goes home.
postponed until further notice.
largely dependent upon support from management.


Closing:
I won't be trapped by dogma - which is living with the results of other people's thinking.
Unfortunately, however, I cannot refund the time you've spent reading this report.
Live long and prosper.
Peace out.
Tag, you're it.
Carpe Diem.
Later Vader.
This message will self destruct.
I had better get off the toilet now.

**Reply by alboss** [reply]
These are absolutely brilliant. You have a frightening grasp of the subject. I've added them.

**Comment by rednbenji4** [comment]
Closing:
We're not the droids you're looking for.

**Reply by alboss** [reply]
Also brilliant, and now part of the mix.
